org.chromium.Chromium.yaml

```diff
@@ -0,0 +1,249 @@
+app-id: org.chromium.Chromium
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: chromium
+finish-args:
+  - --require-version=1.8.2
+  - --filesystem=host-etc
+  - --device=all
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --socket=pulseaudio
+  - --socket=x11
+  - --system-talk-name=org.freedesktop.Avahi
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
```

Comment:
Doesn't it work with secrets portal? Recent libsecret does, so if Chromium accesses secret service through libsecret - it should work with portal as well.